templates/zerver/help/rename-stream.md

```diff
@@ -0,0 +1,16 @@
+# Rename a Stream
+
+If you would like to change the name of an existing stream, locate the cog (![cog](/static/images/help/cog.png)) next to **Streams** in your left sidebar. A tooltip labeled **Subscribe, add, or configure streams** will appear.
+
+![Streams Cog](/static/images/help/streams-1.png)
+
+After clicking the cog, a modal with an overview of all existing streams will pop up.
+
+![Streams Overview](/static/images/help/streams-overview.png)
+
+To change the name of a stream, first click on the stream you want to change. After selecting a stream, a list of settings should appear in the righthand panel under **Stream Settings.** Scroll down to the administrator settings area to see the option to change the stream's name.
```

Comment:
Following your previous formatting, you might want to change your sentence to "Scroll down to  **Administrator settings**" or something similar.